SPDY/SPDYSessionManager.m

```diff
@@ -120,8 +123,11 @@ - (void)queueStream:(SPDYStream *)stream
     [_pendingStreams addStream:stream];
     stream.delegate = self;
 
-    NSTimeInterval deferrableInterval = stream.request.SPDYDeferrableInterval;
+    NSTimeInterval fastFailDeferrableInterval = (_fastFailCount > 0) ? ((_lastFastFailTime + kFastFailDeferTime) - CFAbsoluteTimeGetCurrent()) : 0;
+    NSTimeInterval deferrableInterval = fmax(stream.request.SPDYDeferrableInterval, fmin(fastFailDeferrableInterval, kFastFailDeferTime));
```

Comment:
if you use MAX and MIN, the macro will base the comparison on the types of the values (aka, as doubles), but being explicit with fmax/fmin is fine too